### PR TITLE
Adding contextual links for Biblio entity.

### PR DESCRIPTION
**Pull request by RoySegall**
#108

screenshot from 2014-01-16 10 53 19

### CODE REVIEW

File: modules/biblio_ui/biblio_ui.module [hunk 1]
```diff
@@ -403,12 +405,18 @@ function theme_biblio_ui_add_biblio_list($variables) {
...
-  $info = $wrapper->view();
-  $content = $info['biblio'][$wrapper->getIdentifier()];
+function biblio_ui_view_biblio(Biblio $biblio) {
+  $info = $biblio->view();
 
-  return render($content);
+  $info += array(
+    '#theme' => 'biblio',
+    '#entity' => $biblio,
+    '#view_mode' => 'full',
```
Comment (amitaibu) [inline modules/biblio_ui/biblio_ui.module]: is it always `full`?

**Comment by RoySegall** [comment]
@amitaibu The view mode is now a parameter(like in the node_view function).

File: modules/biblio_ui/biblio_ui.module [hunk 2]
```diff
@@ -402,13 +404,27 @@ function theme_biblio_ui_add_biblio_list($variables) {
...
 /**
  * Viewing a biblio entity entry.
+ *
+ * @param Biblio $biblio
+ *  The biblio entity object.
+ * @param $view_mode
+ *  The view mode. Default to full.
+ *
+ * @return
+ *  The biblio object.
```
Comment (amitaibu) [inline modules/biblio_ui/biblio_ui.module]: I don't think this return description is correct.

File: biblio.module [hunk 3]
```diff
@@ -990,3 +990,55 @@ function biblio_check_biblio_migration_needed() {
...
+  );
+
+  return $info;
+}
+
+/**
+ * Implements hook_biblio_view_alter().
+ */
+function biblio_biblio_view_alter(&$build) {
+  $build['#theme'] = 'biblio';
```
Comment (amitaibu) [inline biblio.module]: why do we do this in the alter -- can't we do it in the `public function view()` or `buildContent()`?

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**